recipe/meta.yaml

```
@@ -30,12 +29,10 @@ requirements:
     # Require `cmake-no-system` to break circular dependency;
     # the `cmake` package on defaults requires `zstd`.
     - cmake-no-system
-    - patch  # [unix]
-    - m2-patch  # [win]
   host:
-    - zlib
-    - lz4-c
-    - xz
+    - zlib {{ libzlib }}
```

```suggestion
    - zlib {{ zlib }}
```

@cbouss since we now have _libzlib_ is it a better idea to use it as @vkhomits did in this recipe or should we still use _{{ zlib }}_?

---

This is fine.

---

Yeah, I would even say it should be either `zlib {{ zlib  }}` or `libzlib {{ libzlib }}`. But the question if to use `libzlib` over `zlib` still remains. I would say, when `zlib` was used before, it should remain as is. Is my thinking right?